doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add the `--filterchpi` option to `mne browse_raw`, by `Felix Raimundo`_
```

Comment:
From digging a little bit, a sphinx link gets created as `_gen_mne_browse_raw`, so you should be able to do:
```
... in :ref:`mne browse_raw <gen_mne_browse_raw>` ...
```
Can you check with `make html_dev-noplot` locally if it works?